pulsar-broker/src/test/java/org/apache/pulsar/broker/transaction/TransactionTest.java

```diff
@@ -802,7 +802,7 @@ public void testCancelTxnTimeout() throws Exception{
     public void testNotChangeMaxReadPositionAndAddAbortTimesWhenCheckIfNoSnapshot() throws Exception {
         PersistentTopic persistentTopic = (PersistentTopic) getPulsarServiceList().get(0)
                 .getBrokerService()
-                .getTopic(NAMESPACE1 + "/test", true)
+                .getTopic(NAMESPACE1 + "/changeMaxReadPositionAndAddAbortTimes", true)
```

Comment:
you can add some suffix (UUID.randomUUID) in order to not reuse the same topic in case of test that is re-run due to a failure